### ISSUE TITLE
Code lacks copyright/license information

### ISSUE THREAD

**Issue by mzmcbride** [monsur/enable-cors.org#54]
I looked around the source code, but couldn't find any copyright/license info.

**Comment by monsur** [comment]
Michael, did the original site have an copyright info?

**Comment by mhausenblas** [comment]
Nope, I don't think so. Same as with #53 please feel free to choose whatever you think makes sense as I don't have any strong opinion on this.

**Comment by indolering** [comment]
AGPL 3.0 or CC0 or (if you REALLY care about software patents) Apache.
